cmd/flow_capture.go

```diff
@@ -336,11 +345,11 @@ func scanner() {
 		if err != nil {
 			panic(err)
 		}
-		if key == keyboard.KeyCtrlC {
+		if key == keyboard.KeyCtrlC || stopReceived {
```

Comment:
If keyboard ctrl+c is a "normal exit" from the TUI, shouldn't it still return a success?

Reply:
That's right, these should be success
https://github.com/netobserv/network-observability-cli/pull/25/commits/01d96f2afe9fdef61212bccf2df5ed1111d69a8c

However I still get a return error in the tests:
```
time="2024-04-12T16:47:23+02:00" level=info msg="Terminating command..." component=capture_test
    capture_test.go:54: 
        	Error Trace:	/home/julien/dev/me/network-observability-cli/e2e/capture_test.go:54
        	            				/home/julien/dev/me/network-observability-cli/vendor/sigs.k8s.io/e2e-framework/pkg/env/env.go:428
        	            				/home/julien/dev/me/network-observability-cli/vendor/sigs.k8s.io/e2e-framework/pkg/env/env.go:447
        	Error:      	Expected nil, but got: &exec.ExitError{ProcessState:(*os.ProcessState)(0xc0003e0078), Stderr:[]uint8(nil)}
```

https://github.com/netobserv/network-observability-cli/pull/25/files#diff-c79801ce4280c1cacebdcb5d72f542e41d73222870a63df8c46ed019689531d2R54-R55